Makefile

```diff
@@ -38,10 +38,12 @@ run: fmt vet ## Run against the configured Kubernetes cluster in ~/.kube/config
 	go run ./main.go
 
 .PHONY: install
+install: export KUBECONFIG = $(KIND_KUBECONFIG)
```

Comment:
This would make it impossible to easily install the CRDs in a non-kind cluster using this Makefile.

I suggest to rather create a `kind-install` target:

```Makefile
.PHONY: kind-install
e2e-install: export KUBECONFIG = $(KIND_KUBECONFIG)
e2e-install: install ## Install the CRDs into the e2e test cluster
```

Reply:
argh, this change was here due to a PEBKAC issue, this should not be in here :)